dist/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <title>Todo List</title>
+    <link rel="stylesheet" href="./style.css" />
+    <script defer="defer" src="index.bundle.js"></script>
+  </head>
+  <body>
+    <header hidden>
+      <nav class="nav">
+        <div><a href="#">Todo List</a></div>
+        <div>
+          <a id="listBtn" class="nav-border" href="#List">List</a>
+          <a id="AddNewBtn" class="nav-border" href="#Add-new">Add new</a>
+        </div>
+      </nav>
+    </header>
+    <main>
+      <section id="todo-section">
+        <h2 class="bg">
+          Today's To Do
+          <button class="btn-no" type="button" id="add-btn">
+            <img id="reloadBtn" src="#" alt="reload" />
+          </button>
+        </h2>
+        <form class="bg">
+          <input
+            id="description"
+            name="description"
+            required
+            placeholder="Add to your list..."
+          />
+          <button class="btn-no" type="submit" id="add-btn">
+            <img id="enterBtn" src="#" alt="submit" />
+          </button>
+        </form>
+        <div id="todo-list">
+          <div class="todo">
+            <div class="w-auto">
+              <input type="checkbox" /> <label>wash the dish</label> <input />
+            </div>
+            <div class="w-auto"><img src="" alt="" srcset="" /></div>
+          </div>
+        </div>
+        <div class="text-center bg" id="clear">
+          <label>Clear all completed</label>
+        </div>
+      </section>
+      <footer></footer>
```

Comment:
It would be best to use your footer section outside the main element. It would be better for HTML semantic tags.